lib/browser/browser.js

```diff
@@ -73,7 +73,9 @@ module.exports = class Browser {
             sessionId,
             ...sessionOpts,
```

Comment:
are we really need `sessionOpts` here now?

Reply:
Yeah, they are used to correctly detect backend configuration, here - https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/index.ts#L86.

Without this session does not attached when used webdriver protocol.